stripe/resource.py

```diff
@@ -936,6 +946,11 @@ def transfers(self, **params):
         return transfers
 
 
+# This resource can only be instantiated when expanded on a BalanceTransaction
+class RecipientTransfer(APIResource):
```

Comment:
Hah, sorry for the continue churn, but I just noticed that like in stripe-ruby, this can and probably should inherit `StripeObject` and not `APIResource`.

Other than that, LGTM.